examples/real-world/index.js

```diff
@@ -1,12 +1,17 @@
 import 'babel-polyfill'
 import React from 'react'
 import { render } from 'react-dom'
+import { browserHistory } from 'react-router'
 import Root from './containers/Root'
 import configureStore from './store/configureStore'
+import { syncHistoryWithStore } from './syncHistoryWithStore'
 
 const store = configureStore()
+const history = syncHistoryWithStore(browserHistory, store, {
+  adjustUrlOnReplay: true
```

Comment:
Note: if you pass `false` **DevTools history time travel will still work in UI**, it just won't update the address bar.